### ISSUE TITLE
Splitting @keyframes leads to wrong behaviour

### ISSUE THREAD

**Issue by nicolashoby** [VitaliyR/postcss-esplit#5]
The default behaviour when the splitting point is within an at-rule, as illustrated by the example in the readme, is to duplicate the at-rule. However, when this happens within an `@keyframes `rule, the result is invalid:
```css
/* input - style.css */
@keyframes myAnimation {
  0% {
    background-color: red;
  }
  100% {
    background-color:transparent
  }
}
```
The result is:
```css
/* style.css */
@keyframes myAnimation {
  0% {
    background-color: red;
  }
}

/* style-0.css */
@keyframes myAnimation {
  100% {
    background-color:transparent
  }
}
````
which is no longer valid.

In this case, the whole `@keyframes` block should be copied to the new file.

**Comment by VitaliyR** [comment]
@nicolashoby Hi! Thanks for finding this one, I've fixed it in v0.1.2. From now, this at-rule block wouldn't be separated.

**Comment by nicolashoby** [comment]
Hi Vitaliy

Thanks very much for the quick fix, much appreciated!

Greetings
Nico
